src/xdocs/config_filters.xml

```diff
@@ -991,6 +991,216 @@ public class InputTest {
       </subsection>
     </section>
 
+    <section name="SuppressionXpathSingleFilter">
+      <subsection name="Description" id="SuppressionXpathSingleFilter_Description">
+        <p>Since Checkstyle 8.16</p>
+        <p>
+          Filter <code>SuppressionXpathSingleFilter</code> suppresses audit events for
+          Checks violations in the specified file, class, method or variable.
```

Comment:
Taken from Javadoc:
> This filter processes {@link TreeWalkerAuditEvent}
objects based on the criteria of files, checks, message, module id, xpathQuery.

You say `method or variable` when it should be like javadoc, `checks, message, module id, and xpath`.

Also some of my points were missed from https://github.com/checkstyle/checkstyle/pull/6172#discussion_r240048432 .
> XpathFilter isn't visible to users, SuppressionXpathFilter is so it would make more sense to users to mention that filter instead of internal one.

My original complaint was I didn't want the technical text in the first sentence. I didn't mean for it all to be removed. Start the users off slowly with easy talk, then as they grasp a basic understanding start making it more complex.

Reply:
> You say `method or variable` when it should be like javadoc, `checks, message, module id, and xpath`.

Sorry sir, not quite understanding the meaning of these words. Is it means that to write what javadoc said inside the description?

Reply:
change `Checks violations in the specified file, class, method or variable` => `Checks violations in the specified file, class, checks, message, module id, and xpath`.
This is what you wrote in the Javadoc, so it should be the same in the XDoc. There is no property method/variable, so the way the XDoc was written was slightly misleading.

Reply:
Sir, I cannot think about the more complex text now. I think my original technical words cannot make users understand what exactly this filter could do. I am not sure except of the words above, what else should I put into the description?